addons/l10n_in/models/template_in.py

```diff
@@ -91,7 +91,7 @@ def _get_l10n_in_fiscal_tax_vals(self, use_zero_rated_igst=False, use_sez_exp_ig
         return [Command.clear()] + [
             Command.create({
                 'tax_src_id': f"sgst_{tax_type}_{rate}",
-                'tax_dest_id': f"igst_{tax_type}_{rate if not use_zero_rated_igst else 0}{'_sez_exp' if use_sez_exp_igst else ''}",
+                'tax_dest_id': f"igst_{tax_type}_{rate if not use_zero_rated_igst else 0}{'_sez_exp' if use_sez_exp_igst and tax_type == 'sale' else ''}",
             })
             for tax_type in ["sale", "purchase"]
```

Comment:
@hamo-odoo Should not we filter here instead?

Reply:
No I don't think so it would work well, the reason is we want the regular purchase taxes for SEZ to be linked and also we have the use_zero_rated_igst. The xml id changes for that. Though the line crosses above 99 character, but the f-string works well here.

Reply:
@hamo-odoo It won't create the same lines twice?  (but maybe they are filtered out...)

Reply:
@jco-odoo I'm not sure what are you thinking about? I don't see lines getting created twice.
![image](https://github.com/user-attachments/assets/5aa4ca09-151e-4177-b05a-074273b97279)

Reply:
ok, it will just fallback to normal interstate.